src/Native/Window.js

```diff
@@ -8,8 +8,16 @@ var size = _elm_lang$core$Native_Scheduler.nativeBinding(function(callback)	{
 	}));
 });
 
+var scrollTo = function(x, y) {
+  return _elm_lang$core$Native_Scheduler.nativeBinding(function(callback)	{
+    window.scrollTo(x, y);
+    callback(_elm_lang$core$Native_Scheduler.succeed());
+  });
+};
+  
 return {
+  scrollTo: F2(scrollTo),
```

Comment:
This is not indented properly, also since scrollTo comes after size, IMHO it must be after that